lusidtools/cocoon/utilities.py

```diff
@@ -940,3 +941,30 @@ def strip_whitespace(df: pd.DataFrame, columns: list) -> pd.DataFrame:
         )
 
     return stripped_df
+
+
+def create_scope_id(time_generator=None):
+    """
+    This function creates a unique ID based on the time since epoch for use
+    as a scope id.
+
```

Comment:
What is time_generator, should it be added to docs?

Reply:
Good find, yes I will add it!